README.md

```diff
@@ -1,5 +1,16 @@
 # PyPSA-Earth-Sec: A Sector-Coupled Open Optimisation Model of the Global Energy System
 
-## Development Status: Under Development
+## Development Status: Version 0.0.5
 
-PyPSA-Earth-Sec is currently under development.
+Disclaimer: PyPSA-Earth-Sec is still under development.
+
+The model currently covers one country, Morocco, in a simplistic representation with only 4 nodes (simplistic). A prerequisite for a successful model run is built network generated by pypsa-africa. Currently, no real demand data is used for the country inspected, instead, we use dummy data that is either taken from a different country relying on the data used in pypsa-eur-sec or generated using heuristics.
+
+The model now includes the following energy carriers: \n
+        **Electricity**, **Hydrogen**, **Fossil gas**, **Fossil oil** and **Carbon**.
+
+The demand sectors covered are: **residential**, **industry**, **land transport**, **aviation** and **shipping** 
+
+The diagram below depicts one representative clustered node showing the combination of carriers and sectors covered in the model as well as the generation and conversion technologies included. 
+
+![alt text](https://github.com/pypsa-meets-africa/pypsa-earth-sec/blob/Readme-for-MVP/docs/0.0.5_wb.png?raw=true)
```

Comment:
Remarks to the graphs:
In the H2 node, "Land transport fuel" is listed. Since it only includes H2 (and not other fuels like oil, which are in the liquid fossil copper plated carriers), I think it's a little bit misleading and could be changed to "H2 for land transport". What do you think?